softgroup/basis/common-router/router-impl/pom.xml

```diff
@@ -3,29 +3,34 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>services</artifactId>
+        <artifactId>common-router</artifactId>
         <groupId>com.softgroup</groupId>
         <version>0.0.1</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>main-router</artifactId>
+    <artifactId>router-impl</artifactId>
 
     <dependencies>
         <dependency>
             <groupId>com.softgroup</groupId>
+            <artifactId>router-api</artifactId>
+            <version>0.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.softgroup</groupId>
             <artifactId>authorization-impl</artifactId>
-            <version>${project.version}</version>
+            <version>0.0.1</version>
```

Comment:
I need this dependency for properly injecting FirstRouter , FirstRouter has Factory , Factory has other other Router , each Router has Factory , each Factory has RequestHandlers, and all these classes must be injected by spring , without these dependency It is not possible